examples/realworld/README.md

```diff
@@ -8,13 +8,6 @@ Here, we implement it in Wasp, by following their [specification](https://realwo
 This app is deployed at https://wasp-rwa.netlify.app/ .
 
 # Development
-
-### Database
```

Comment:
If we are removing this block, why not mention `wasp start db` instead?

Reply:
Good point, added!